examples/common/java/direct/worker/gradlew.bat

```diff
@@ -1,4 +1,18 @@
 @if "%DEBUG%" == "" @echo off
+@rem Copyright 2020 Intel Corporation
```

Comment:
I don't think Intel created files under java folder

Reply:
OK--I removed the Copyright line.